src/blocks/pagination/Pagination.tsx

```diff
@@ -0,0 +1,106 @@
+import Pagination from 'rc-pagination';
+import styled from 'styled-components';
+import 'rc-pagination/assets/index.css';
+import { FC } from 'react';
+import { CaretLeft, CaretRight } from 'blocks/icons';
+import { getTextVariantStyles } from 'blocks/Blocks.utils';
+import { PaginationProps } from './Pagination.types';
+import enUS from 'rc-pagination/lib/locale/en_US';
+
+const StyledPagination = styled(Pagination)<{ disabled?: boolean }>`
```

Comment:
1. The three dots size and color and border-radius doesn't matches the one in figma.
2. The caret icon color in dark mode are broken and doesn't matches figma.

Figma:

<img width="274" alt="Screenshot 2024-08-26 at 12 43 00 PM" src="https://github.com/user-attachments/assets/14c36c2a-fe97-4701-aa4d-53213771d3a2">

Usage in DApp:
<img width="1102" alt="Screenshot 2024-08-26 at 12 43 21 PM" src="https://github.com/user-attachments/assets/950c028a-4a7c-408d-9523-807bd5f2255c">